java/s2i/bin/s2i-setup

```diff
@@ -0,0 +1,5 @@
+# Local vars setup with defaults
+S2I_DESTINATION=${S2I_DESTINATION:-/tmp}
+S2I_SOURCE_DIR="${S2I_DESTINATION}/src"
+S2I_ARTIFACTS_DIR="${S2I_DESTINATION}/artifacts"
+DEPLOYMENTS_DIR="/opt/app-root/src/"
```

Comment:
Note, s2i-setup is a copy of an env file bundled with fabric8 that we overwrite because otherwise there is no way to change the DEPLOYMENTS_DIR env (it's sourced in the assemble script).

We change this here so that we can generate the JAR directly into the src dir